### PR TITLE
Flexible map choices

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/grda_warehouse/hud/project_coc.rb [hunk 1]
```diff
@@ -55,6 +55,12 @@ class ProjectCoc < Base
       where(arel_table[:City].lower.in(place.map(&:downcase)))
     end
 
+    scope :in_county, ->(county:) do
+      county = Array(county)
+      # FIXME
```
Comment (awisema) [inline app/models/grda_warehouse/hud/project_coc.rb]: What needs fixing?

File: drivers/public_reports/app/models/public_reports/setting.rb [hunk 2]
```diff
@@ -9,6 +9,16 @@ class Setting < GrdaWarehouseBase
     attr_encrypted :s3_access_key_id, key: ENV['ENCRYPTION_KEY'][0..31]
     attr_encrypted :s3_secret, key: ENV['ENCRYPTION_KEY'][0..31]
 
+    def self.available_map_types
+      types = {
+        coc: 'Continuum of Care',
+        # county: 'County',
```
Comment (awisema) [inline drivers/public_reports/app/models/public_reports/setting.rb]: Should this be enabled?

File: app/models/grda_warehouse/hud/project_coc.rb (hunk 1 above)
Comment (eanders) [reply app/models/grda_warehouse/hud/project_coc.rb]: So, we need to actually figure out how to calculate county (which is why the next comment you had is disabled.)  Holding off on merging this until we have that settled.

File: drivers/public_reports/app/models/public_reports/setting.rb (hunk 2 above)
Comment (eanders) [reply drivers/public_reports/app/models/public_reports/setting.rb]: Commented out until we know how to calculate county for a project.

**Review by awisema — APPROVED** [review]

**Pull request merged**